source/index.html

```diff
@@ -1,12 +1,162 @@
 <!DOCTYPE html>
-<html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Кэт энерджи</title>
-  </head>
-  <body>
+<html class="page" lang="ru">
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+
+<body class="page-body">
+  <header class="page-header">
+    <div class="page-header__container">
+      <div class="page-header__logo-container">
+        <a class="page-header__logo-image" href=""></a>
+        <button class="page-header__logo-toggle"></button>
+      </div>
+      <nav class="page-header__nav site-nav">
+        <ul class="site-nav__list">
+          <li class="site-nav__item">
+            <a class="site-nav__link site-nav__link--active" href="">главная</a>
+          </li>
+          <li class="site-nav__item">
+            <a class="site-nav__link" href="">Каталог продукции</a>
+          </li>
+          <li class="site-nav__item">
+            <a class="site-nav__link" href="">Подбор программы</a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <main class="page-main">
+    <section class="page-main__promo promo">
+      <div class="promo__container">
+        <h1 class="promo__title">Функциональное питание для котов</h1>
+        <p class="promo__slogan">Занялся собой? Займись котом!</p>
+        <div class="promo__picture"></div>
+        <a class="promo__program-choose" href="">подобрать программу</a>
+      </div>
+    </section>
+    <section class="programs">
+      <h2 class="visually-hidden">Программа питания для котов</h2>
+      <div class="program__container">
+        <ul class="program__list">
+          <li class="program__item">
+            <h3 class="program__title">похудение</h3>
+            <p class="program-discription">
```

Comment:
Думаю пока лучше сделать через элемент, а не отдельный блок, т.к. сам `p` не содержит в себе элементов, и отдельный блок ему не нужен
```suggestion
            <p class="program__description">
```